modules/docs/arrow-docs/docs/docs/typeclasses/hash/README.md

```diff
@@ -51,6 +51,11 @@ val user = User("MyId", "MyName")
 
 // This is fine
 Hash.any().run { user.hash() }
+```
+
+```kotlin:ank
+data class User(val id: String, val name: String)
```

Comment:
Same as with the imports, there's no need to redefine the class either!

Reply:
The ones that are playable which if possible should be most of the ones used to document API require all the context in which they run in them

Reply:
Ok I'll change that then. But that only goes for context as in classes and variables, not imports right?

Reply:
See the `arrow-query-language` docs for examples. You have to include everything in the snippet that it needs to run. All imports, a main method and a couple of comments that delimit the code you are not collapsing in the snippet when presented on the site.

Reply:
Done (I hope) :)